flocker/testtools/__init__.py

```diff
@@ -807,3 +811,35 @@ class CustomException(Exception):
     An exception that will never be raised by real code, useful for
     testing.
     """
+
+
+class _ProcessResult(PRecord):
+    """
+    The return type for ``run_process`` representing the outcome of the process
+    that was run.
+    """
+    command = field(type=list, mandatory=True)
+    output = field(type=bytes, mandatory=True)
+    status = field(type=int, mandatory=True)
+
+
+def run_process(command, *args, **kwargs):
+    """
+    Run a child process, capturing its stdout and stderr.
+
+    :param list command: An ``argv`` to use to launch the child process.
```

Comment:
:question: This is a little bit confusing to me but might be Python terminology I've just not come across - it reads like `argv` is the type. A quick search's first results are all about `sys.argv` and don't really seem to fit with "an argv".